virttest/libvirt_xml/network_xml.py

```diff
@@ -371,7 +371,7 @@ class NetworkXMLBase(base.LibvirtXMLBase):
                  'autostart', 'persistent', 'forward', 'mac', 'ip',
                  'bandwidth_inbound', 'bandwidth_outbound', 'portgroup',
                  'dns', 'domain_name', 'nat_port', 'forward_interface',
-                 'routes', 'virtualport_type')
+                 'routes', 'virtualport_type', 'mtu')
```

Comment:
I believe `mtu` entry have to be added in `class Interface('network')` in virttest/libvirt_xml/devices/interface.py

https://libvirt.org/formatdomain.html#mtu

Reply:
Yes, and mtu entry also exists in network.